src/main/java/org/aksw/simba/squirrel/analyzer/impl/Guess.java

```diff
@@ -0,0 +1,38 @@
+package org.aksw.simba.squirrel.analyzer.impl;
+
+public class Guess {

```

Comment:
A class that has the same name as the interface above is not a very good idea :disappointed: